lemur_vault/plugin.py

```diff
@@ -300,6 +302,69 @@ def authenticate_certificate():
         raise Exception('Vault Config: cert or key path not set.')
 
 
+def generate_gcp_jwt():
+    """
+    Generate GCP JWT for Vault authentication.
+    :return: GCP JWT
+    """
+    role = current_app.config.get('VAULT_AUTH_ROLE')
+    account = current_app.config.get('VAULT_AUTH_ACCOUNT')
+
+    if role and account:
+        headers = {'Metadata-Flavor': 'Google'}
+        url = 'http://metadata/computeMetadata/v1/instance/service-accounts/' + account + '/identity'
```

Comment:
It looks like this code is assuming that it is running on a GCE instance. If this is the case then this could be simplified to just the application default service account.

https://cloud.google.com/compute/docs/instances/verifying-instance-identity

```
url = 'http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/identity'
```

This will use the service account that the GCE instance is running as without having to explicitly specify the name of the service account.